nncf/quantization/algorithms/bias_correction/onnx_backend.py

```diff
@@ -43,7 +43,7 @@
 class ONNXBiasCorrectionAlgoBackend(BiasCorrectionAlgoBackend):
     @property
     def tensor_processor(self) -> ONNXNNCFCollectorTensorProcessor:
-        return ONNXNNCFCollectorTensorProcessor()
+        return ONNXNNCFCollectorTensorProcessor
```

Comment:
Have you tested it? I see no changes in the `algorithm` class to create `tensor_processor` instead of it's call.